advanced_functionality/pytorch_deploy_large_GPT_model/GPT-J-6B-model-parallel-inference-DJL.ipynb

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
<p>Unfinished sentence? <code>Models such as  </code></p><p> </p><p>Standardize: is it <code>Deepspeed</code> or <code>DeepSpeed</code>?</p><p> </p><p>Please use present tense throughout the notebook: <code>In this notebook, we distributed</code> -&gt; <code>... we distribute</code></p><p> </p><p>You say here that the notebook will use a p4d.24xlarge instance, but later you use a ml.g5.48xlarge instance - be consistent.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3540/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
resolved unfinished sentence, settled on DeepSpeed, went to present tense for all but the conclusion (which is past tense: in this notebook you did xyz), switched to g5 instance